### PR TITLE
M2-811: [Mobile] Migrate Flanker for Android

### PR DESCRIPTION
**Pull request by anq83**
M2-811: [Mobile] Migrate Flanker for Android

### CODE REVIEW

File: src/entities/flanker/lib/types/configuration.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  buttonLabel: string;
+  minimumAccuracy: number;
+  continueText: Array<string>;
+  restartText: Array<string>;
+};
+
+export type StimulusScreen = {
+  id: StimulusConfigId;
+  correctChoice: number;
+  stimulus: { en: string };
```
Comment (vmkhitaryanscn) [inline src/entities/flanker/lib/types/configuration.ts:67]: Does this need to be updated to stimulus: string from backend?

Comment (anq83) [reply src/entities/flanker/lib/types/configuration.ts:67]: Unfortanetelly this contract for the existing html page which is working with PsichoJs library.

File: src/entities/flanker/ui/mocks/FlankerWithImageOnButton.ts [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+  showFixation: true,
+  showFeedback: true,
+  showResults: true,
+  samplingMethod: 'fixed-order',
+  nextButton: 'OK',
+  sampleSize: 1,
+  trialDuration: 5000,
+  fixationDuration: 1000,
+  fixationScreen: {
+    value: 'Без названия.png',
```
Comment (BamMironov) [inline src/entities/flanker/ui/mocks/FlankerWithImageOnButton.ts:111]: Are you sure about that?

File: src/entities/flanker/ui/HtmlFlanker.tsx [hunk 3]
```diff
@@ -0,0 +1,101 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+/* eslint-disable react-native/no-inline-styles */
```
Comment (BamMironov) [inline src/entities/flanker/ui/HtmlFlanker.tsx:2]: Why is it necessary to disable them?

**Review by BamMironov — CHANGES_REQUESTED** [review]

File: src/entities/flanker/ui/mocks/FlankerWithImageOnButton.ts (hunk 2 above)
Comment (anq83) [reply src/entities/flanker/ui/mocks/FlankerWithImageOnButton.ts:111]: Yes, just a mock, but based on real data taken from old app api

**Pull request merged**